app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,10 @@
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  included do
```

Comment:
@eclarizio why do we need this, rails already handles this 
ActionDispatch::ExceptionWrapper.rescue_responses accounts for a record not found
https://github.com/rails/rails/blob/b2d8036df7427dca67d1cbe1e3c3e0fd7b33bf56/activerecord/lib/active_record/railtie.rb#L22

Reply:
Good question, but the JIRA issue exists, and when I wrote a spec it ends up blowing up without returning a 404, so it apparently isn't working.

@syncrou Any idea?

Reply:
@mkanoor @eclarizio There are other ways to do this using middleware. The reason we need to pass a status and a render: the `ActionDispatch::ExceptionWrapper.rescue_responses` works at the `request` layer and this error raised after the request was passed down into the controller method. 

I agree we shouldn't need to do the above, but without capturing the status at the routes layer in middleware there is no way to know what action should be taken. We can define it above or as a `rescue` in the method where the error happened ( like we do for a number of `create` methods.